orochi/client.py

```diff
@@ -20,6 +20,7 @@
 from .colors import bold, title
 from .xdg import get_orochi_xdg_dir
 from . import meta
+from os.path import expanduser
```

Comment:
I'm already importing `os` on line 4, so you could just use `os.path.expanduser`. But you won't need it, I'll tell you why in another comment.